src/json2daisy/json2daisy.py

```diff
@@ -190,6 +190,12 @@ def generate_header(board_description_dict: dict) -> 'tuple[str, dict]':
   else:
     replacements['display'] = ''
 
+  if target['defines'].get('OOPSY_TARGET_HAS_MIDI_INPUT'):
+    replacements['midi'] = """
```

Comment:
We could split UART and USB between different PD "ports" (so that one is "channels" 1-16 and the other 17-32 inside the patch). Or make this configurable somehow.